packages/frontend/src/retro/components/dialogs/TimerDialog.tsx

```diff
@@ -19,43 +20,14 @@ export function TimerDialog({
   remainingMinutes,
   remainingSeconds,
 }: TimerDialogProps) {
-  const {
-    retroState,
-    handleStartTimer,
-    handleStopTimer,
-    handlePauseTimer,
-    handleResumeTimer,
-    handleChangeTimer,
-  } = useRetroContext();
+  const { retroState, handleStartTimer, handleStopTimer, handlePauseTimer, handleResumeTimer } =
+    useRetroContext();
   const { timerStatus } = retroState;
   const isTimerRunning = timerStatus === TimerStatus.RUNNING;
   const isTimerPaused = timerStatus === TimerStatus.PAUSED;
   const isTimerStopped = timerStatus === TimerStatus.STOPPED;
-
-  const {
-    value: seconds,
-    formattedValue: formattedSeconds,
-    isError: isSecondsError,
-    onChange: handleSecondsChange,
-  } = useValidatedTimeInput({
-    formatLength: 2,
-    maxValue: 59,
-    minValue: 0,
-    initialValue: 0,
-  });
-
-  const {
-    value: minutes,
-    formattedValue: formattedMinutes,
-    isError: isMinutesError,
-    onChange: handleMinutesChange,
-    incrementTime: incrementMinutes,
-  } = useValidatedTimeInput({
-    formatLength: 2,
-    minValue: 0,
-    maxValue: 99,
-    initialValue: 5,
-  });
+  const [minutes, setMinutes] = useState(remainingMinutes);
+  const [seconds, setSeconds] = useState(remainingSeconds);
```

Comment:
This state is always changed together, therefore it should be combined.

  const [time, setTime] = useState<{minutes, seconds}>({minutes: remainingMinutes, seconds: remainingSeconds});